static/js/components/downloads.js

```diff
@@ -5,6 +5,30 @@
   var catRef = null
   var caretOffsetRef = null
 
+  var getLatestVersion = (function() {
+    var cache = {}
+
+    return function(slug, callback) {
+      if (cache.hasOwnProperty(slug)) {
+        callback(null, cache[slug])
+      }
+
+      $.get('https://img.resin.io/api/v1/image/' + slug + '/versions', function(response) {
+        var version = response.latest.replace(/.prod/, '.dev');
```

Comment:
I think that we should escape the dot in this regex.

Reply:
If-you-are-in-the-mood: Someone could argue that we should use resin-semver to order the array of returned versions and pick the latest `.dev` edition from there, instead of using a replace. We had an issue on the Dashboard for something similar I think, but it might be an overkill for this case.

Reply:
Or the JSON returned by e.g. https://img.resin.io/api/v1/image/raspberry-pi2/versions could have both a `latest` and a `latest_dev` key?

Reply:
@lurch I'm wondering what other clients would be able to use that information - it's certainly helpful.

@thgreasi Implementing `resin-semver` here is a little tricky, as the JS build on this site is a bit primitive and AFAIK `resin-semver` is designed to run in a commonJS environment. The code I'm using here is lifted from https://github.com/resin-os/resinos/blob/master/config/updateDeviceVersions.js which is what has been used historically to update the OS versions. I agree about escaping the `.` in the RegEx btw